docs/ABOUT.md

```diff
@@ -1,5 +1,23 @@
-C# is a modern, object oriented language developed and maintained by Microsoft. The learning curve is small, reduced even if the developer has experience with other 'C-Style' languages.
+C# is a multi-paradigm statically-typed programming language with object-oriented, declarative, functional, generic, lazy integrated querying features and type inference. 
 
-C# is a carefully designed language with lots of great features, such as type-inference and async/await. The official [documentation](https://docs.microsoft.com/en-us/dotnet/csharp/) is very extensive and well-written.
+__Statically-typed__ means that identifiers have a type set at compile time--like those in Java, C++ or Haskell--instead of holding data of any type like those in Python, Ruby or JavaScript.
 
-Although .NET used to be Windows-only, with the release of [.NET Core](https://www.microsoft.com/net/core) you can also use C# on Mac or Unix-based systems.
+__Object-oriented__ means that C# provides imperative class-based ojects with features such as single inheritance, interfaces and encapsulation.  
+
+__Declarative__ means programming what is to be done, as opposed to how it is done (which is an implementation detail which can distract from the domain or business logic).
+
+__Functional__ means that functions are first-class data types that can be passed as arguments to and returned from other functions.
+
+__Generic__ means that algorithms are written in terms of types to-be-specified-later that are then instantiated when needed for specific types provided as parameters. 
+
+__Lazy__ (a.k.a "non-strict") means that the compiler will put off evaluating a thing until absolutely neccessary. This lets you safely do weird stuff like operating on an infinite list--the language will only create it up to the last value you actually use.
+
+__Integrated Querying__ means the language feature called [LINQ "Language-Integrated Query"](https://msdn.microsoft.com/en-us/library/bb308959.aspx), which enables querying directly within the language, not only its own objects but, also, external data sources through formats such as XML, Json, SQL, NoSQL DBs and event streams.  
```

Comment:
Json -> JSON

Reply:
and will add to "enables querying" "enables lazy querying"?

Reply:
Added deferred instead of non-strict and a couple of minor tweaks to the language (only on bits copied from the Haskell page)